tests/unit_tests/test_ustar_cp/test_computeStandardisedScores.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import numpy as np
+
+@pytest.mark.parametrize(
+    "input_data, expected_xNormX",
+    [
+        #
+        # Test 1: 2x3 matrix, no NaNs
+        #
+        #   x = [[1,2,3],
+        #        [4,5,6]]
+        #
+        #   For each column (with only 2 values):
+        #     median => (value1 + value2)/2
+        #     iqr    => 75th - 25th percentile => typically 1.5 for each col
+        #
+        #   => xNorm => row0 => [-1, -1, -1]
+        #               row1 => [ 1,  1,  1]
+        #   => xNormX => [1, 1]
+        #
+        (
+            [
+                [1, 2, 3],
+                [4, 5, 6]
+            ],
+            [[np.nan],[np.nan]],
+        ),
+
+        #
+        # Test 2: 3x3 matrix, no NaNs
+        #
+        #   x = [[1,2,3],
+        #        [4,5,6],
+        #        [7,8,9]]
+        #
+        #   column 0 => median=4, iqr=3
+        #   column 1 => median=5, iqr=3
+        #   column 2 => median=6, iqr=3
+        #
+        #   => xNorm => row0 => [-1, -1, -1]
+        #               row1 => [ 0,  0,  0]
+        #               row2 => [ 1,  1,  1]
+        #   => xNormX => [1, 0, 1]
+        #
```

Comment:
```suggestion
        # Test 2: 3x3 matrix, no NaNs
```
Comments removed as inconsistent with test definition